### ISSUE TITLE
Blazing Herald

### ISSUE THREAD

**Issue by ACG8** [ACG8/Lucid#217]
![blazing herald](https://user-images.githubusercontent.com/10094820/43983119-88530fa2-9cbe-11e8-8766-23f8751309e0.jpg)

**Comment by millerj-tc** [comment]
If interpreted during the dive phase, this would break the convention that there are no card effects that persist once the card leaves limbo and is buried. I don't see a compelling reason to break that convention at this point.

We could either table it for an expansion with permanents/"until end of round" effects or reword it so that the paid ability is conditional on interpreting it during the dream phase.

**Comment by ACG8** [comment]
What if it were a Lucid instead?

**Comment by ANRTheBigBoy** [comment]
Ya this is too weird for now.

**Comment by ACG8** [comment]
Works much better as a lucid with orb rules:

![blazing herald](https://user-images.githubusercontent.com/10094820/44236164-a667d880-a171-11e8-9d9f-5e7eb51575bb.jpg)

**Comment by ANRTheBigBoy** [comment]
I like the top ability. "Prepare your top card" is already an on-par ability for the burst effect, and this is a lot better. Let's rework the burst to something else. 

One idea: you are going to be putting down double dreams with this, so your hand will likely have dives/lucids in it. How about [bury dive][bury lucid]: Gain [i4]

Another issue (that I've raised before) is that you don't actually "interpret" dreams in the dream phase. You just reveal them and do what they say. Reword this to "You may dream 1 additional time".

**Comment by ACG8** [comment]
![blazing herald](https://user-images.githubusercontent.com/10094820/44633365-fa33a800-a94f-11e8-8286-a3ddaa7a417e.jpg)

**Comment by ANRTheBigBoy** [comment]
The Interp bonus is wrong.

**Comment by ACG8** [comment]
![blazing herald](https://user-images.githubusercontent.com/10094820/44752917-20894d00-aae2-11e8-8acb-9e207d9ad9cb.jpg)
